lib/python/ensembl/brc4/runnable/manifest_stats.py

```diff
@@ -168,13 +195,68 @@ def parse_gff3(self, gff3_handle: TextIO) -> List:
             sorted_biotypes[name] = data
         
         stats = [f"{data['unique_count']:>9}\t{biotype:<20}\tID = {data['example']}" for (biotype, data) in sorted_biotypes.items()]
+
+        # Check against NCBI stats
+        stats += self.check_ncbi_stats(biotypes,self.param('accession'))
         
         return stats
+    
+    def check_ncbi_stats(self, biotypes: Dict, accession: str) -> List:
+        """Use the dataset tool from NCBI to get stats and compare with what we have"""
+
+        stats = []
+
+        datasets_bin = self.param('datasets_path')
+        if not which(datasets_bin):
+            return stats
+        
+        # Get the dataset summary from NCBI
+        command = [datasets_bin, "summary", "genome", "accession", accession]
+        result_out = subprocess.run(command, stdout=subprocess.PIPE)
+        result = json.loads(result_out.stdout)
+
+        # Get stats
+        if "reports" in result:
+            genome = result["reports"][0]
+            if "annotation_info" in genome and "stats" in genome["annotation_info"]:
+                ncbi_stats = genome["annotation_info"]["stats"]
+
+                if "gene_counts" in ncbi_stats:
+                    counts = ncbi_stats["gene_counts"]
+                    stats += self.compare_ncbi_counts(biotypes, counts)
```

Comment:
Since the list should be empty at this point, I believe you can get away with less "compute":
```suggestion
                    stats = self.compare_ncbi_counts(biotypes, counts)
```